pokedex/data/csv/items.csv

```diff
@@ -303,98 +303,98 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
 302,reaper-cloth,10,2000,10,
 303,razor-claw,12,5000,80,
 304,razor-fang,12,5000,30,7
-305,tm01,37,5000,,
-306,tm02,37,3000,,
-307,tm03,37,3000,,
-308,tm04,37,3000,,
-309,tm05,37,3000,,
-310,tm06,37,3000,,
-311,tm07,37,3000,,
-312,tm08,37,3000,,
-313,tm09,37,3000,,
-314,tm10,37,10000,,
-315,tm11,37,3000,,
-316,tm12,37,5000,,
+305,tm01,37,40000,,
```

Comment:
lol, i guess that's what happens when TMs reuse the same items every game. we really need a way to track cost changes. i noticed a bunch of other items had their cost changed too.